static/app/views/replays/details.tsx

```diff
@@ -53,11 +34,21 @@ class ReplayDetails extends AsyncView<Props, State> {
   getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
     const {params, location} = this.props;
 
-    const eventView = EventView.fromLocation(location);
-    const query = eventView.getEventsAPIPayload(location);
+    const eventList = EventView.fromSavedQuery({
+      id: '',
+      name: '',
+      version: 2,
+      fields: ['timestamp', 'replayId'],
+      orderby: 'timestamp',
+      projects: [],
+      range: '14d',
+      query: `transaction:sentry-replay`,
+    });
+    eventList.additionalConditions.addFilterValues('replayId', [params.replayId]);
+    const apiPayload = eventList.getEventsAPIPayload(location);
 
     return [
-      ['event', `/organizations/${params.orgId}/events/${params.eventSlug}/`, {query}],
```

Comment:
So we'll still want this because this refers to the `sentry-replay` transaction. All rrweb attachments are uploaded to this event only (and none of the `sentry-replay-event` transactions).

Reply:
We talked offline and this is indeed correct as we can have multiple `sentry-replay` transactions with the same replayId (e.g. when page is reloaded we'll need to create a new transaction but with the same replayId).